src/locale/en.json

```
@@ -931,7 +931,7 @@
   },
   "Settings": {
     "AutoLockTagged": "Sync item lock state with tag",
-    "AutoLockTaggedExplanation": "DIM will automatically lock and unlock items to match their tag. When this setting is enabled, the lock icon will not be shown on the item tile for tagged items.",
+    "AutoLockTaggedExplanation": "DIM will automatically lock and unlock items to match their tag. When this setting is enabled, the lock icon will not be shown on the item tile for tagged items. Excludes crafted items.",
```

```suggestion
    "AutoLockTaggedExplanation": "DIM will automatically lock and unlock items to match their tag. Crafted items will remain unlocked to allow reshaping. When this setting is enabled, the lock icon will not be shown on the item tile for tagged items.",
```
rather not drop out of full english sentences for this one headline-style thought.